### ISSUE TITLE
Text nodes only write into depth buffer when a render state node is connected

### ISSUE THREAD

**Issue by tebjan** [mrvux/dx11-vvvv#336]
in order to interact with the depth buffer, the text nodes need for example a Blend node connected.

**Comment by mrvux** [comment]
Indeed, default text render state ignores depth buffer, this will be changed (even though it can be a small breaking change)
